### ISSUE TITLE
E/SchedAssist : ofbSetSfMsgTrans failed with error=Permission denied

### ISSUE THREAD

**Issue by blinoff82** [OnePlusOSS/android_kernel_oneplus_sm8350#21]
OnePlus 9rt, OOS 13.1
Build number: MT2111_13.1.0.500(EX01)
Kernel: 5.4.210-qgki-g7faff7b28df2

Logcat is completely occupied by thousands of identical lines:
06-12 18:04:52.843 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.851 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.868 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.876 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.884 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.892 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.900 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.909 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied
06-12 18:04:52.917 E/SchedAssist( 1549): ofbSetSfMsgTrans pid(1549) failed with error=Permission denied

**Comment by OnePlusOSSAdmin** [comment]
The log point would be invokded frequently in SurfaceFlinger. It seems that the process doesn't get right permission. Could you close selinux temporarily to check whether these logs would appear again?

**Comment by blinoff82** [comment]
Yes, admin. I set "setenforce 0" via Termux and the error is gone. 
Maybe sepolicy, maybe we need to chown for sched assist nodes (if I'm not mistaken, this is your addition to CAF perf)

**Comment by fanf** [comment]
I confirm both the log overflow and the change when disabling selinux with `setenforce 0`

**Comment by min38514** [comment]
I have the same problem and I don't know why this error occurs. Is it because of magisk?

**Comment by blinoff82** [comment]
No.

**Comment by min38514** [comment]
Only setenforce 0 is solution?

**Comment by Validbit** [comment]
Could this be brought to attention, or how to identify what process/app is triggering this? I had this on my non-oneplus device (RMX3311) three updates back (several months) and it may've gorten worse (as if the severe spamming wasn't bad enough)
